nix/haskell.nix

```diff
@@ -20,6 +20,7 @@ let
       name = "cardano-node";
       src = ../.;
     };
+    hpack = null;
```

Comment:
:question: 

This needs explanation with a comment.

Reply:
I don't know why this is in there. Dropped, let's see what happens.